main.go

```diff
@@ -32,6 +32,7 @@ func main() {
 
 	flagSet.String("http-address", "127.0.0.1:4180", "[http://]<addr>:<port> or unix://<path> to listen on for HTTP clients")
 	flagSet.String("https-address", ":443", "<addr>:<port> to listen on for HTTPS clients")
+	flagSet.Bool("force-https", false, "force HTTPS redirect")
```

Comment:
Potential improvement?
```suggestion
	flagSet.Bool("force-https", false, "force HTTPS redirect for HTTP requests")
```